src/goto-instrument/code_contracts.cpp

```diff
@@ -968,3 +1002,731 @@ bool code_contractst::enforce_contracts(
   }
   return fail;
 }
+
+/*********************************************
+ *  Assigns Clause Target - Scalar
+ ********************************************/
```

Comment:
Not our way of formatting comments, and also not particularly informative (it's exactly the same as the name of the class).